smokeTest.js

```diff
@@ -0,0 +1,112 @@
+const XETHRU_STATE = require('./SessionStateXethruEnum.js')
+const MOV_THRESHOLD = 17
+const IM21_DOOR_STATUS = require('./IM21DoorStatusEnum')
+const testLocation1Id = 'TestLocation1'
```

Comment:
I think by having this hardcoded we can only ever run one smoketest at a time, right? I think that's OK, but it might be worth noting in the README.

Reply:
True, do you think it's worth generating a unique id here to avoid this and allow arbitrary smoke tests?

Reply:
I think it's OK just having the one. Although maybe change the name to something more obviously smoketest-related so that if it ever does end up in the DB then it'll be clear that it can just be deleted.